autogpt/utils.py

```diff
@@ -10,24 +10,12 @@
 # Use readline if available (for clean_input)
 try:
     import readline
-except:
+except ImportError:
     pass
 
 from autogpt.config import Config
 
 
-def send_chat_message_to_user(report: str):
```

Comment:
This function does nothing.  `can_handle_report` is not an attribute of the plugin template so this literally always returns without doing anything.

Reply:
this is a new change to the template, it does indeed work